blocks/carousel/carousel.css

```diff
@@ -193,6 +208,20 @@ main .section div.carousel-wrapper {
 }
 
 /* Media queries */
+
+/* Tablet */
+@media screen and (min-width: 768px) {
+    .carousel.image-carousel-full-width .carousel-slide-container {
+        gap: 61px;
+    }
+
+    .carousel.image-carousel-full-width.no-animation-effect .carousel-slide-container {
+        grid-template-columns: 90px 90px 90px 90px;
```

Comment:
Where is this `90px` column width coming from? I'm not seeing these specs anywhere within the current Stat Bar pattern (https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?type=design&node-id=1461%3A13718&mode=dev).

Reply:
This 90px is actually the width of the logos on tablet view. as we are using CSS Grid to align and provide width to the cards, we need to add this for less than 4 cards scenario.
<img width="1464" alt="Screenshot 2023-08-31 at 3 29 58 PM" src="https://github.com/hlxsites/merative2/assets/135624814/40cfc67d-daa7-4580-9de0-1f7348e1c4b1">